website/docs/reference/node-selection/methods.md

```diff
@@ -73,6 +73,19 @@ selectors unambiguous.
 
 </VersionBlock>
 
+<VersionBlock firstVersion="1.2">
+
+### The "file" method
+The `file` method can be used to select a model by its filename, including the file extension (`.sql`).
```

Comment:
worth noting that if you make a model file with a different extension than `.sql` you cannot select it with this syntax? e.g.
`dbt run some_model.xlsx`

so the change is mostly just ergonomic? relatedly, before I could pass the absolute filename ending with `.sql` (inclusive of path), but I could never before just name a file with `.sql`? TIL

Reply:
> worth noting that if you make a model file with a different extension than .sql you cannot select it with this syntax?

it also would not be parsed / understood as a dbt model: [`.sql` only](https://github.com/dbt-labs/dbt-core/blob/f988f76fccc1878aaf8d8631c05be3e9104b3b9a/core/dbt/parser/read_files.py#L154) (soon `.py`, too!)

> so the change is mostly just ergonomic? relatedly, before I could pass the absolute filename ending with `.sql` (inclusive of path), but I could never before just name a file with `.sql`? TIL

exactly right!